kbfsmd/root_metadata_v2_test.go

```diff
@@ -162,106 +161,6 @@ func (wmef writerMetadataExtraV2Future) toCurrent() WriterMetadataExtraV2 {
 	return wmef.WriterMetadataExtraV2
 }
 
-// TODO: Remove the below once we move this file to kbfsmd.
-
-type tlfCryptKeyInfoFuture struct {
-	kbfsmd.TLFCryptKeyInfo
-	kbfscodec.Extra
-}
-
-func (cki tlfCryptKeyInfoFuture) toCurrent() kbfsmd.TLFCryptKeyInfo {
-	return cki.TLFCryptKeyInfo
-}
-
-func (cki tlfCryptKeyInfoFuture) ToCurrentStruct() kbfscodec.CurrentStruct {
-	return cki.toCurrent()
-}
-
-func makeFakeTLFCryptKeyInfoFuture(t *testing.T) tlfCryptKeyInfoFuture {
```

Comment:
Why are these helpers being removed?

Reply:
They're duplicates of stuff already in kbfsmd -- my previous PR temporarily had them in both kbfsmd and libkbfs.